tap_google_ads/__init__.py

```diff
@@ -12,14 +12,11 @@
 
 REQUIRED_CONFIG_KEYS = [
     "start_date",
-    "oauth_client_id",
-    "oauth_client_secret",
-    "refresh_token",
-    "customer_ids",
     "developer_token",
+    "impersonated_email",
+    "auth_method"
```

Comment:
We do not necessarily need to require these fields to be in the `REQUIRED_CONFIG_KEYS` set of fields.